### ISSUE TITLE
ASK: Multiple KV Read Regions

### ISSUE THREAD

**Issue by coderbuzz** [denoland/deploy_feedback#729]
When a PRO member activates another read region(s), are the KV read units multiplied by as many the read regions?

**Comment by losfair** [comment]
No, amount of KV read units consumed is not affected by the number of read regions.

**Comment by coderbuzz** [comment]
> No, amount of KV read units consumed is not affected by the number of read regions.

Thank you for your quick feedback

Do you have plan to allow PRO members to configure write regions?
